libass/ass_render.c

```diff
@@ -3049,6 +3141,78 @@ ass_render_event(RenderContext *state, ASS_Event *event,
     return true;
 }
 
+#if ENABLE_THREADS
+static ASSThreadReturn ASS_THREAD_FUNC_ATTR run_thread(void* ptr)
+{
+    ASS_Renderer *priv = ptr;
+
+    TextInfo text_info = {0};
+    RenderContext state = {
+        .renderer = priv,
+        .text_info = &text_info,
+    };
+
+#define THREAD_NAME "libass/render"
+    assi_set_thread_name("libass/render");
```

Comment:
It seems as though you wanted to use the macro here or to drop the macro definition.

Reply:
Yup, forgot to drop.